libs/@blockprotocol/hook/README.md

```diff
@@ -38,7 +38,7 @@ We also provide a `useHook` hook to make sending hook messages easier.
 ```typescript
 import { useHook } from "@blockprotocol/hook/react";
 
-useHook(hookService, nodeRef, "text", "$.text", (node) => {
+useHook(hookService, nodeRef, "text", ["text"], (node) => {
```

Comment:
We should make this and other examples (e.g. in `test-react-block.tsx`) into BaseURIs if we can now

Reply:
<img width="437" alt="image" src="https://user-images.githubusercontent.com/25749103/219368303-7fb20092-0150-455c-af49-b8a3f3a262b6.png">

Hook service fixed [Update hook service usages](https://github.com/blockprotocol/blockprotocol/pull/979/commits/afc325cdaa109c7d5a591d203704c8722a6af1b4)